features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64LRUCacheTest.kt

```diff
@@ -140,13 +147,12 @@ internal class Base64LRUCacheTest {
     @Test
     fun `M not generate key prefix W put() { layerDrawable with only one layer }`(
         @Mock mockRippleDrawable: RippleDrawable,
-        @Mock mockBgLayer: Drawable,
-        @StringForgery fakeBase64: String
+        @Mock mockBgLayer: Drawable
```

Comment:
nit: better not to shorten

```suggestion
        @Mock mockBackgroundLayer: Drawable
```